packages/dev/core/src/Rendering/fluidRenderer/fluidRendererGUI.ts

```diff
@@ -0,0 +1,453 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { KeyboardEventTypes } from "core/Events/keyboardEvents";
+import type { KeyboardInfo } from "core/Events/keyboardEvents";
+import { Color3 } from "core/Maths/math.color";
+import type { Observer } from "core/Misc/observable";
+import type { Scene } from "core/scene";
+import type { Nullable } from "core/types";
+import { Tools } from "core/Misc/tools";
+
+import { FluidRenderingDebug } from "./fluidRenderingTargetRenderer";
+
+async function LoadLiLGUI(): Promise<void> {
+    return Tools.LoadScriptAsync("https://cdn.jsdelivr.net/npm/lil-gui@0.17.0/dist/lil-gui.umd.min.js");

```

Comment:
we try to avoid embedding anything directly, but if it is still needed - we should also allow changing this URL so that devs could host it themselves. Also need to be sure that esm devs would still be able to use this.
If it is a package, maybe we should consider adding it as an optional dependency for the public repository. But we can talk about it :-)

Reply:
Yes, as I said above:

`
If we don't want to reference the LiLGUI package from FluidRenderingGUI, maybe I should remove the class and add it in the documentation for users as a debugging tool?
`
It would probably be the best thing to do? This way, the users can still copy/paste the debug class from the doc and we don't have to update the inspector just now.

Reply:
Class being removed.